### DISCUSSION TITLE
Batch delete consecutive files.

### DISCUSSION THREAD

**Discussion by tttoad** [nvim-tree/nvim-tree.lua#2677] [Q&A]
My project generates some consecutive files, and when I want to delete them I always need to delete them one by one.
Select consecutive files in V mode and press "d" to delete them all at once.
It would be great if there was support for deleting selected non-contiguous files.

**Comment by alex-courtis** [comment]
You can bulk delete them: mark them all with `m` then `bd`.

`:help nvim-tree-mappings-default`
`:help nvim-tree-api.marks`
